src/lib.rs

```diff
@@ -31,6 +31,7 @@ pub struct Router {
     cache_config_path: PathBuf,
     module_cache: PathBuf,
     default_host: String,
+    http_concurrency: Option<u32>,
```

Comment:
Could this be `http_max_concurrency`, or something a bit more explicit?

Reply:
I agree here. This should be more specific

Reply:
done